### PR TITLE
cleaning up filesystem errors on build-static

### PR DESCRIPTION
**Pull request by davidmsibley**
changes to frame weren't showing up when I ran static:dev, so I went through and fixed the build script.  

Standardized error throwing, copy now overwrites.

----

Contributor License Agreement adherence:

- [x] This Contribution is under the terms of Individual [Contributor License Agreements][] (and also Corporate Contributor License Agreements to the extent applicable) appearing in the [Apereo CLA roster][].

[Apereo CLA roster]: http://licensing.apereo.org/completed-clas
[Contributor License Agreements]: https://www.apereo.org/licensing/agreements

### CODE REVIEW

File: tools/uw-frame-static/build.js [hunk 1]
```diff
@@ -12,9 +12,12 @@ var exec_handler = function(error, stdout, stderr) {
...
 var mkdirp = require('mkdirp');
 var copy = require('recursive-copy');
+var copyOptions = {
+  overwrite: true
+}
 
-mkdirp('uw-frame-static/target/css/themes/', function (err) {
-  if (err) throw error;
+mkdirp('uw-frame-static/target/css/themes/', function (error) {
+  if (error) throw error;
```
Comment (ChristianMurphy) [inline tools/uw-frame-static/build.js]: could `error` be confused with `Error` here?

File: tools/uw-frame-static/build.js [hunk 2]
```diff
@@ -12,9 +12,12 @@ var exec_handler = function(error, stdout, stderr) {
 
 var mkdirp = require('mkdirp');
 var copy = require('recursive-copy');
+var copyOptions = {
```
Comment (ChristianMurphy) [inline tools/uw-frame-static/build.js]: assuming Node.js 4+ is the target run time, should this be a `const`?

File: tools/uw-frame-static/build.js (hunk 1 above)
Comment (davidmsibley) [inline tools/uw-frame-static/build.js]: I want to trust that `Error` would be immediately noticeable due to classes starting with capitals by convention.  If someone saw `Error` and not `Error()` or `new Error()` that should be picked up during review.

File: tools/uw-frame-static/build.js (hunk 2 above)
Comment (davidmsibley) [inline tools/uw-frame-static/build.js]: yup. and we're using const elsewhere in the file.  I'll update the file to use it consistently.

File: tools/uw-frame-static/build.js (hunk 1 above)
Comment (ChristianMurphy) [inline tools/uw-frame-static/build.js]: Got it, you are expecting Google code style's [`new-cap`](http://eslint.org/docs/rules/new-cap) lint rule to catch this.

**Pull request merged**